tests/config.py

```diff
@@ -93,3 +93,7 @@
 # Whether or not to perform some options as a thread. Turning this off may help
 # with debugging at the cost of performance.
 _USE_THREADS = os.getenv('ROOKCHECK_USE_THREADS', True)
+
+# Whether or not to use the ansible "free" strategy. It is generally faster
+# but causes the output to be intertwined and more confusing.
+_USE_FREE_STRATEGY = os.getenv('ROOKCHECK_USE_FREE_STRATEGY', False)
```

Comment:
I wonder if we want this here. Looks like this can be already configured via `ansible.cfg`. See  https://docs.ansible.com/ansible/latest/user_guide/playbooks_strategies.html

Reply:
We're not using/supplying any ansible.cfg afaik (unless I'm mistaken and Ansible is still loading from /etc/ somewhere..). We might use it in the future though.

Nevertheless, we are explicitly setting the strategy type for each playbook run, so I think it is needed. (At least until we change to using yaml playbooks).